src/errors/location.rs

```diff
@@ -143,6 +143,12 @@ impl ToPyObject for Location {
     }
 }
 
+impl IntoPy<PyObject> for Location {
+    fn into_py(self, py: Python<'_>) -> PyObject {
+        self.to_object(py)
+    }
+}
+
```

Comment:
Not 100% sure this is necessary.

Reply:
i don't think this is necessary, you should be able to use `location.to_object(py)`?